languages/go/exercises/concept/basics/.docs/after.md

```diff
@@ -0,0 +1,101 @@
+# Packages
```

Comment:
I know this is still draft so I won't review it properly, but to save you the pain for later, all headings should be `##`. 
@ErikSchierboom I'm not sure if we have documented this anywhere?

Reply:
Was wondering about that already. Thanks for letting me know.

Reply:
We do: https://github.com/exercism/v3/blob/master/docs/maintainers/style-guide.md#headers